securedrop_client/logic.py

```diff
@@ -109,6 +109,12 @@ class Client(QObject):
     """
     authentication_state = pyqtSignal(bool)
 
+    """
+    This signal indicates that a file has been successfully downloaded by emitting the file's
+    UUID as a string.
+    """
+    file_downloaded = pyqtSignal(str)
```

Comment:
We may want to name this `file_ready` like we do for the similar signals in `{Message,Reply}Sync`. Do we want this signal to emit if a download completes by decryption fails?

Reply:
Yeah `file_ready` is a better name since we're only emitting this signal when the file both download/decrypts. In terms of handling decryption failures, i.e. attempting to decrypt again, I think we should tackle in #82 (the scope of this PR is just to resolve the regressions)